account-kit/react/src/hooks/useUser.ts

```diff
@@ -35,6 +39,13 @@ export const useUser = (): UseUserResult => {
   const eoaUser = useMemo(() => {
     if (account.status !== "connected" && account.status !== "reconnecting") {
       return null;
+    } else if (account.status == "reconnecting") {
```

Comment:
⚠️ **[eslint]** <[eqeqeq](https://eslint.org/docs/latest/rules/eqeqeq)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected '===' and instead saw '=='.
<!-- __reviewdog__:ChBjYjcxNWRmNTNmMWRjMWJkEgZyZGpzb24= -->

Reply:
thanks @moldy530 . I am brand new to ts. 

change is on its way.